### ISSUE TITLE
Display an alt text over the language flags

### ISSUE THREAD

**Issue by chandonn-zz** [brenoalvs/monk#191]
When an user hovers over a language flag, it could show an alt text with its name. This might increase readability and some screen readers can use this feature too, in order to tell the idiom of the post in question

**Comment by brenoalvs** [comment]
@chandonn does this work with SVG files?
Our flags are in SVG.

**Comment by chandonn-zz** [comment]
Unfortunately alt attribute does not work. But in order to improve our accessibility, the way is using `title` and `desc` inside svg declaration

**Comment by chandonn-zz** [comment]
According to the conventions for web accessibility, the best way is to add the `aria-label` attribute in the construction of `span` tags.
Please, reference here: https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html
